tests/cephfs/cephfs_vol_management/cephfs_vol_mgmt_subvolgroup_octal_modes.py

```diff
@@ -0,0 +1,392 @@
+import logging
+import random
+import string
+import traceback
+
+from ceph.parallel import parallel
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+
+log = logging.getLogger(__name__)
```

Comment:
Can you use Log.

from utility.log import Log
log = Log(__name__)

Instead of 
log = logging.getLogger(__name__)